src/frontend/src/pages/FinancePage/ReimbursementRequestsSection.tsx

```diff
@@ -25,6 +27,9 @@ const ReimbursementRequestTable = ({
   userReimbursementRequests,
   allReimbursementRequests
 }: ReimbursementRequestTableProps) => {
+  const [order, setOrder] = useState<SortingOrder>(SortingOrder.ascending);
```

Comment:
Since we know that the sorting order is either ascending or descending, let's set the data type of the order to be a boolean that indicates if it's an ascending order. For ex, we can then name the variable as `[isAscendingOrder, setAscendingOrder]`.

Apply this change in the rest of this file and in the types file.